org.geany.Geany.appdata.xml

```diff
@@ -28,18 +28,19 @@ as possible from a specific Desktop Environment like KDE or GNOME.</p>
   </description>
   <screenshots>
     <screenshot type="default">
-      <image height="808" width="942">http://www.geany.org/uploads/Gallery/geany_main.png</image>
+      <image width="1366" height="741">http://www.geany.org/uploads/Gallery/geany_main.png</image>
```

Comment:
- Use HTTPS if possible.
- Are the width and height fields necessary?

Reply:
No idea, just changed it because the CI system complained about a mismatch between the xml and the png.

Reply:
You can remove them, failure was due to _mismatch_ with previous image.